deps/internal/cmds/get/topology.go

```diff
@@ -164,6 +285,28 @@ func topologyCommand(
 				return err
 			}
 
+			if !(destinationModuleName == "") {
```

Comment:
If users don't pass in a destination module, the cmd behaves as it is now

Reply:
hmm. I would put this in it's own command... something like `deps get paths -l go --from a [--to b]`

Reply:
In that case, should we look at the dependencies (or downstream) or dependents (or upstream) or both to find all paths?

Reply:
If we think about it from the perspective of why users would want all paths between modules, I think, looking at `dependencies` paths by default makes sense?

Reply:
I think dependencies make sense to me. The use case here is often to answer the question of "how does Application X pull in Library Y?" Eventually, we'll want to do things like follow versions on edges but that's definitely a follow-on feature.